llama_stack/apis/agents/agents.py

```diff
@@ -195,12 +196,63 @@ class AgentToolGroupWithArgs(BaseModel):
 register_schema(AgentToolGroup, name="AgentTool")
 
 
+from llama_stack.apis.inference import (
+    ToolType as InferenceToolType,
+    WebSearchTool,
+    WolframAlphaTool,
+    CodeInterpreterTool,
+    FunctionToolDefinition,
+)
+
+
+class ToolType(InferenceToolType):
+    knowledge_search = "knowledge_search"  # currently rag tool
+
+    # MCP
+    model_context_protocol = "model_context_protocol"
```

Comment:
could shorten to `mcp` even given a newborn will also know what it is, it appears